recipes/server.rb

```diff
@@ -66,3 +66,13 @@
 sysctl_param 'vm.swappiness' do
   value 0
 end
+
+cookbook_file '/usr/local/bin/mysql-accounting' do
+  source 'mysql-accounting'
+  mode '0755'
+end
```

Comment:
Is there a downside to copying a file over?  It might be nice to have if we decide we want to run the script manually.